.github/workflows/docker-release.yml

```
@@ -0,0 +1,153 @@
+name: Docker stable release
+
+on:
+  release:
+    types:
+      - released
+      - prereleased
+
+jobs:
+  ssr:
+    name: Build SSR docker image
```

```suggestion
    name: Build SSR image
```

Docker is already part of the workflow, no need to repeat